packages/amplication-client/src/Workspaces/WorkspaceLayout.tsx

```diff
@@ -95,6 +101,7 @@ const WorkspaceLayout: React.FC<Props> = ({ innerRoutes, moduleClass }) => {
     workspaceId: currentWorkspace?.id,
     projectId: currentProject?.id,
     resourceId: currentResource?.id,
+    version,
```

Comment:
This adds the version to all events under the workspace but it will not add the version to other events. 
Instead, I think we should register the version number in the init function of the analytics service and use it on all track calls

Reply:
isn't it part of the analytics track event ?